src/rpc/rawtransaction.cpp

```diff
@@ -1084,12 +1084,12 @@ static UniValue sendrawtransaction(const JSONRPCRequest& request)
 {
     if (request.fHelp || request.params.size() < 1 || request.params.size() > 2)
         throw std::runtime_error(
-            "sendrawtransaction \"hexstring\" ( allowhighfees )\n"
+            "sendrawtransaction \"hexstring\" ( allowhighfees|maxfeerate )\n"
             "\nSubmits raw transaction (serialized, hex-encoded) to local node and network.\n"
             "\nAlso see createrawtransaction and signrawtransaction calls.\n"
             "\nArguments:\n"
             "1. \"hexstring\"    (string, required) The hex string of the raw transaction)\n"
-            "2. allowhighfees    (boolean, optional, default=false) Allow high fees\n"
+            "2. allowhighfees|maxfeerate (boolean/numeric, optional, default=false/∞) Allow high fees (boolean) or reject transactions whose fee rate is higher than the specified value (numeric), expressed in " + CURRENCY_UNIT + "/kB\n"
```

Comment:
Shouldn't this default be whatever `maxTxFee` is, not `∞`?

Reply:
Fixed - it now outputs
```
2. allowhighfees|maxfeerate (boolean/numeric, optional, default=false/0.10) Allow 
high fees (boolean) or reject transactions whose fee rate is higher than the 
specified value (numeric), expressed in BTC/kB
```
for the default settings.